article/articles.py

```diff
@@ -1,42 +1,112 @@
 from collections import namedtuple
 
 from directory_constants.constants import exred_articles
-from django.core.urlresolvers import reverse_lazy
+from django.urls import reverse_lazy
 
 
 Article = namedtuple(
     'Article',
     ['uuid', 'title', 'keywords', 'tasks', 'markdown_file_path', 'url',
      'parent']
 )
-ArticleParent = namedtuple('ArticleParent', ['uuid', 'title'])
+ArticleParent = namedtuple('ArticleParent', ['uuid', 'title', 'articles'])
 
 
 GUIDANCE_MARKET_RESEARCH = ArticleParent(
     uuid='GUIDANCE_MARKET_RESEARCH',
     title='Market research',
+    articles=frozenset((
+        exred_articles.DO_RESEARCH_FIRST,
+        exred_articles.DEFINE_MARKET_POTENTIAL,
+        exred_articles.DO_FIELD_RESEARCH,
+        exred_articles.ANALYSE_THE_COMPETITION,
+        exred_articles.VISIT_TRADE_SHOW
+    ))
 )
```

Comment:
these are already defined in `structure.py` - see `GUIDANCE_MARKET_RESEARCH_ARTICLES` etc (the `ArticleGroup`).

Note `ArticleParent` is just for displaying this the name of the category the article is within e.g., "Market research".

![image](https://user-images.githubusercontent.com/5485798/31996023-c1a29760-b97e-11e7-93cf-f9251cdcbd46.png)

I can see the naming is confusing. Perhaps we should instead remove `ArticleParent` do something like this:

```
CATEGORY_LABEL_MARKET_RESEARCH = 'Market research'

DO_RESEARCH_FIRST = Article(
    ...
    category_label=CATEGORY_LABEL_MARKET_RESEARCH,
)
DEFINE_MARKET_POTENTIAL = Article(
    ...
    category_label=CATEGORY_LABEL_MARKET_RESEARCH,
)
...


```